lib/install/bin/webpack-dev-server.tt

```diff
@@ -23,20 +26,39 @@ end
 begin
   dev_server = YAML.load_file(CONFIG_FILE)[RAILS_ENV]["dev_server"]
 
-  DEV_SERVER_HOST = "http#{"s" if args('--https') || dev_server["https"]}://#{args('--host') || dev_server["host"]}:#{args('--port') || dev_server["port"]}"
+  public_host = args('--host') || dev_server["host"]
+  port        = args('--port') || dev_server["port"]
+  https       = ARGV.include?('--https') || dev_server["https"]
 
 rescue Errno::ENOENT, NoMethodError
   puts "Webpack dev_server configuration not found in #{CONFIG_FILE}."
   puts "Please run bundle exec rails webpacker:install to install webpacker"
   exit!
 end
 
+begin
+  server = TCPServer.new(LISTEN_IP_ADDR, port)
+  server.close
+rescue Errno::EADDRINUSE
+  $stdout.puts "Another program is running on port #{port}. Enter a new port number(ex: #{port + 1})"
+  port = $stdin.gets.chomp.to_i
+  retry
```

Comment:
I'm -1 on prompting for a new port. It only temporarily solves the problem and you'll likely encounter it each time you run `bin/webpack-dev-server`. I'd rather print instructions for changing the port in `webpacker.yml` and bail.